core/src/main/java/com/graphhopper/routing/util/parsers/BikeCommonPriorityParser.java

```diff
@@ -180,7 +180,8 @@ void collect(ReaderWay way, double wayTypeSpeed, TreeMap<Double, Integer> weight
         String cycleway = way.getFirstPriorityTag(Arrays.asList("cycleway", "cycleway:both", "cycleway:left", "cycleway:right"));
         if (SHARED_CYCLEWAYS.contains(cycleway)) {
             weightToPrioMap.put(100d, UNCHANGED.getValue());
-        } else if ("lane".equals(cycleway) || "track".equals(cycleway)) {
+        } else if ("lane".equals(cycleway) || "track".equals(cycleway)
+                || "opposite_lane".equals(cycleway) || "opposite_track".equals(cycleway)) {
```

Comment:
Here I would keep the `opposite_lane` as unchanged. I read a report that accident numbers here are pretty high and this sounded reasonable to me. But slightlig boosing `opposite_track` sound reasonable, I would give this a SLIGHT_PREFER.